src/components/Firestore/index.tsx

```diff
@@ -120,54 +115,52 @@ export const Firestore: React.FC<FirestoreProps> = ({ config, projectId }) => {
     <Spinner span={12} data-testid="firestore-loading" />
   ) : (
     <FirestoreStore>
-      <ApiProvider value={api}>
-        <GridCell span={12} className="Firestore">
-          <div className="Firestore-actions">
-            <CustomThemeProvider use="warning" wrap>
-              <Button unelevated onClick={() => handleClearData(api)}>
-                Clear all data
-              </Button>
-            </CustomThemeProvider>
-          </div>
-          <Elevation z="2" wrap>
-            <Card className="Firestore-panels-wrapper">
-              <InteractiveBreadCrumbBar
-                base="/firestore"
-                path={path}
-                onNavigate={handleNavigate}
-              />
-              <div className="Firestore-panels">
-                <Root />
-                {/*
-                 * TODO: This should really be handled by the constituent Document/Collection components,
-                 * where those components can conditionally show their nested-child. Doing so will be
-                 * easiest once we move to a Suspense-ful render, where we can always show the
-                 * Document/Collection-skeletons, but lazy-render the lists wihin those components.
-                 */}
-                {showCollectionShell && (
-                  <div
-                    className="Firestore-Collection"
-                    data-testid="collection-shell"
-                  >
-                    <PanelHeader id="" icon={null} />
-                  </div>
-                )}
-                {showDocumentShell && (
-                  <div
-                    className="Firestore-Document"
-                    data-testid="document-shell"
-                  >
-                    <PanelHeader id="" icon={null} />
-                  </div>
-                )}
-              </div>
-            </Card>
-          </Elevation>
-        </GridCell>
-      </ApiProvider>
+      <GridCell span={12} className="Firestore">
+        <div className="Firestore-actions">
+          <CustomThemeProvider use="warning" wrap>
+            <Button unelevated onClick={() => handleClearData()}>
+              Clear all data
+            </Button>
+          </CustomThemeProvider>
+        </div>
+        <Elevation z="2" wrap>
+          <Card className="Firestore-panels-wrapper">
+            <InteractiveBreadCrumbBar
+              base="/firestore"
+              path={path}
+              onNavigate={handleNavigate}
+            />
+            <div className="Firestore-panels">
+              <Root />
+              {/*
+               * TODO: This should really be handled by the constituent Document/Collection components,
```

Comment:
Does this comment still apply? Can we move shell into Document/Collection components now?

Reply:
Ah, the comment no longer applies, _but_ we still need this. Root/Document/Collection/etc still only know about their children. When someone lands on the /firestore module, we still need this to show 3-columns when no Root-collection is selected.